pkg/apis/OWNERS

```diff
@@ -0,0 +1,13 @@
+# Disable inheritance as this is an api owners file
+options:
+  no_parent_owners: true
+filters:
+  ".*":
+    approvers:
+      - api-approvers
+  # examples:
+  #   pkg/apis/*/types.go
+  #   pkg/apis/*/*/types.go
+  "[^/]+/([^/]+/)?(register|types)\\.go$":
+    labels:
+      - kind/api-change
```

Comment:
and a similar file for `config/crds/*.yaml-patch` files.

Reply:
or even `config/crds/*.yaml` too

Reply:
@pweil- this is still missing.

Reply:
updated and ready for your review (on Monday 😄)

changes: updated contrib wording, added alias, added owners files for apis, crds, and admission.  Labels for `apis` follow the upstream example of `(register|types).go`, labels for `admission` just tags any change, labels for `crds` looks for `.yaml`